lua/toggleterm/ui.lua

```diff
@@ -7,13 +7,26 @@ local fn = vim.fn
 local fmt = string.format
 local api = vim.api
 
-local persistent = {}
 local origin_window
 
-function M.save_window_size()
-  -- Save the size of the split before it is hidden
-  persistent.width = vim.fn.winwidth(0)
-  persistent.height = vim.fn.winheight(0)
+persistent = {}
```

Comment:
Without the `local` keyword this variable is now global 😅 , it shouldn't need to be if it is now only referenced in this module

Reply:
Oops, I forgot to add `local`.